bin/http-server

```diff
@@ -118,6 +119,20 @@ function listen(port) {
       cert: argv.C || argv.cert || 'cert.pem',
       key: argv.K || argv.key || 'key.pem'
     };
+    try {
+      fs.lstatSync(options.https.cert);
+    }
+    catch (err) {
+      logger.info(colors.red('Error: Could not stat certificate ' + options.https.cert));
```

Comment:
While this error message is perfectly correct, could we make it a little friendlier by saying something like "Error: Could not find certificate" rather than "stat"?